### PR TITLE
Chunk manager support scalar data

### PR DESCRIPTION
**Pull request by longjiquan**
Signed-off-by: dragondriver <jiquan.long@zilliz.com>
/kind improvement
issue: #15999

### CODE REVIEW

File: internal/storage/utils.go [hunk 1]
```diff
@@ -837,3 +839,56 @@ func GetPkFromInsertData(collSchema *schemapb.CollectionSchema, data *InsertData
...
+	case *Int16FieldData:
+		return binaryWrite(endian, field.Data)
+	case *Int32FieldData:
+		return binaryWrite(endian, field.Data)
+	case *Int64FieldData:
+		return binaryWrite(endian, field.Data)
+	case *FloatFieldData:
+		return binaryWrite(endian, field.Data)
+	case *DoubleFieldData:
+		return binaryWrite(endian, field.Data)
```
Comment (congqixia) [inline internal/storage/utils.go:890]: Nit: maybe use short form like:
```
case *FloatVectorFieldData, *Int8FieldData, *Int16FieldData...
      return binaryWrite(endian, field.Data)
```

Comment (congqixia) [reply internal/storage/utils.go:890]: did not notice it was type cast switch, please ignore this comment

Comment (DragonDriver) [reply internal/storage/utils.go:890]: The types of field is not same here.

**Pull request merged**